spec.md

```diff
@@ -659,12 +659,13 @@ engine and language handles them. When present, they MUST be in the following or
 
 * Inheritance modifier: `abstract` or `final`
 * Visibility modifier: `public`, `protected`, or `private`
+* Set-visibility modifier: `public(set)`, `protected(set)`, or `private(set)`
 * Scope modifier: `static`
 * Mutation modifier: `readonly`
 * Type declaration
 * Name
 
-All keywords MUST be on a single line, and MUST be separated by a single space.
+All keywords MUST be on a single line, and MUST be separated by a single space.  All keywords MUST be all lower-case.  When using a set-visibility on a public-read property, omitting the `public` keyword is RECOMMENDED but NOT REQUIRED.
```

Comment:
`NOT REQUIRED` is not an RFC 2119 keyword so it should be lowercase and probably omitted.
Though I would opt for a `MAY` rather than recommend:

```suggestion
All keywords MUST be on a single line, and MUST be separated by a single space.  All keywords MUST be all lower-case.  The `public` keyword MAY be omitted when using a set-visibility on a public-read property.
```

Though frankly I probably wouldn't advocate omitting the public, the same way I wouldn't omit public when writing public properties today.